diff --git a/lib/config.guess b/lib/config.guess
index 256083a70..40fdd5e96 100755
--- a/lib/config.guess
+++ b/lib/config.guess
@@ -4,7 +4,7 @@
 
 # shellcheck disable=SC2006,SC2268 # see below for rationale
 
-timestamp='2021-06-03'
+timestamp='2022-11-13'
 
 # This file is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -992,19 +992,30 @@ EOF
 	;;
     arm*:Linux:*:*)
 	set_cc_for_build
-	if echo __ARM_EABI__ | $CC_FOR_BUILD -E - 2>/dev/null \
-	    | grep -q __ARM_EABI__
-	then
-	    GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
-	else
-	    if echo __ARM_PCS_VFP | $CC_FOR_BUILD -E - 2>/dev/null \
-		| grep -q __ARM_PCS_VFP
-	    then
-		GUESS=$UNAME_MACHINE-unknown-linux-${LIBC}eabi
-	    else
-		GUESS=$UNAME_MACHINE-unknown-linux-${LIBC}eabihf
-	    fi
-	fi
+	## With GCC 4.9 arm*-*-gnu is gone (see JB#45154), the compiler
+	## test result has changed, and we don't want eabihf because of
+	## (old comment):
+	###  HACK! A temporarily hack. See MER#1218.
+	###  The problem is that our RPM doesn't have macros directory for
+	###  *-eabihf platform. So it can't find macros file at all that
+	###  breaks build of some packages. The correct solution is to upgrade
+	###  RPM version that is a bit complex task. Please remove this hack
+	###  after RPM upgrade.
+	## ...so we just return eabi
+	# if echo __ARM_EABI__ | $CC_FOR_BUILD -E - 2>/dev/null \
+	#     | grep -q __ARM_EABI__
+	# then
+	#     GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
+	# else
+	#     if echo __ARM_PCS_VFP | $CC_FOR_BUILD -E - 2>/dev/null \
+	# 	| grep -q __ARM_PCS_VFP
+	#     then
+	# 	GUESS=$UNAME_MACHINE-unknown-linux-${LIBC}eabi
+	#     else
+	# 	GUESS=$UNAME_MACHINE-unknown-linux-${LIBC}eabihf
+	#     fi
+	# fi
+	GUESS=$UNAME_MACHINE-unknown-linux-${LIBC}eabi
 	;;
     avr32*:Linux:*:*)
 	GUESS=$UNAME_MACHINE-unknown-linux-$LIBC
